address/address_special.go

```diff
@@ -9,14 +9,38 @@ const (
 	// StakingBucketPoolAddr is the staking bucket pool address
 	StakingBucketPoolAddr = "io000000000000000000000000stakingprotocol"
 
+	// StakingCreateAddr is the special address for staking create actions
+	StakingCreateActionAddr = "io000000000000000000000000web3stakecreate"
+
+	// StakingAddDepositActionAddr is the special address for staking add deposit actions
+	StakingAddDepositActionAddr = "io00000000000000000000web3stakeadddeposit"
+
+	// StakingChangeCandidateActionAddr is the special address for staking change candidate actions
+	StakingChangeCandidateActionAddr = "io000000000000000web3stakechangecandidate"
+
+	// StakingReclaimActionAddr is the special address for staking reclaim actions
+	StakingReclaimActionAddr = "io00000000000000000000000web3stakereclaim"
```

Comment:
reclaim is the common base structure for unstake, withdraw
so change Reclaim to unstake, withdraw